<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>طباعة الفتورة </string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QFrame" name="frame_8">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>90</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
border-bottom:2px dotted#111;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <item row="0" column="2">
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <family>Eras Bold ITC</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border:0px;</string>
           </property>
           <property name="text">
            <string>رقم فاتورة  :   5</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Eras Bold ITC</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border:0px;</string>
           </property>
           <property name="text">
            <string>رقم هاتف  :  0778932688</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_9">
           <property name="font">
            <font>
             <family>Eras Bold ITC</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border:0px;</string>
           </property>
           <property name="text">
            <string>تاريخ  :  10 / 15 / 2022</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Eras Bold ITC</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border:0px;</string>
           </property>
           <property name="text">
            <string>اسم مشتري   :  سليم </string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_10">
           <property name="font">
            <font>
             <family>Eras Bold ITC</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border:0px;</string>
           </property>
           <property name="text">
            <string>تصنيف  :  مواد غذائية </string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>Eras Bold ITC</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border:0px;</string>
           </property>
           <property name="text">
            <string>عنوان  :  برج بوعريريج</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="frame_3">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>GulimChe</family>
                <pointsize>24</pointsize>
               </font>
              </property>
              <property name="text">
               <string>فاتورة الشراء </string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tableWidget">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QTableWidget  {
border:1px solid#111;
background-color:#fff;

}
QTableWidget  ::section{
padding:8px;
border:1px solid#555;
background-color:#eee;
}

QTableWidget::item{
background-color:#fff;
border:1px solid#555;
width:150px;
}

QTableWidget QTableCornerButton::section{
background-color:transparent;
border:1px solid#fff;


}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="lineWidth">
               <number>1</number>
              </property>
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="autoScroll">
               <bool>false</bool>
              </property>
              <property name="verticalScrollMode">
               <enum>QAbstractItemView::ScrollPerItem</enum>
              </property>
              <property name="horizontalScrollMode">
               <enum>QAbstractItemView::ScrollPerItem</enum>
              </property>
              <property name="showGrid">
               <bool>false</bool>
              </property>
              <property name="gridStyle">
               <enum>Qt::NoPen</enum>
              </property>
              <property name="cornerButtonEnabled">
               <bool>true</bool>
              </property>
              <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
               <bool>true</bool>
              </attribute>
              <row>
               <property name="text">
                <string>1</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>2</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>3</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>4</string>
               </property>
              </row>
              <column>
               <property name="text">
                <string>#</string>
               </property>
               <property name="toolTip">
                <string/>
               </property>
               <property name="whatsThis">
                <string/>
               </property>
               <property name="font">
                <font>
                 <pointsize>8</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="textAlignment">
                <set>AlignLeading|AlignVCenter</set>
               </property>
              </column>
              <column>
               <property name="text">
                <string>item</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>unit </string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>quantity</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>amout</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>New Column</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>New Column</string>
               </property>
              </column>
              <item row="0" column="0">
               <property name="text">
                <string>1</string>
               </property>
              </item>
              <item row="0" column="1">
               <property name="text">
                <string>phone </string>
               </property>
              </item>
              <item row="0" column="2">
               <property name="text">
                <string>1500</string>
               </property>
              </item>
              <item row="0" column="3">
               <property name="text">
                <string>10</string>
               </property>
              </item>
              <item row="0" column="4">
               <property name="text">
                <string>2500</string>
               </property>
              </item>
              <item row="1" column="0">
               <property name="text">
                <string>2</string>
               </property>
              </item>
              <item row="1" column="1">
               <property name="text">
                <string>cake </string>
               </property>
              </item>
              <item row="1" column="2">
               <property name="text">
                <string>10</string>
               </property>
              </item>
              <item row="1" column="3">
               <property name="text">
                <string>1</string>
               </property>
              </item>
              <item row="1" column="4">
               <property name="text">
                <string>1200</string>
               </property>
              </item>
              <item row="2" column="0">
               <property name="text">
                <string>3</string>
               </property>
              </item>
              <item row="2" column="1">
               <property name="text">
                <string>wordpress</string>
               </property>
              </item>
              <item row="2" column="2">
               <property name="text">
                <string>1200</string>
               </property>
              </item>
              <item row="2" column="3">
               <property name="text">
                <string>1</string>
               </property>
              </item>
              <item row="2" column="4">
               <property name="text">
                <string>1200</string>
               </property>
              </item>
              <item row="3" column="0">
               <property name="text">
                <string>4</string>
               </property>
              </item>
              <item row="3" column="1">
               <property name="text">
                <string>pc mac book </string>
               </property>
              </item>
              <item row="3" column="2">
               <property name="text">
                <string>1500</string>
               </property>
              </item>
              <item row="3" column="3">
               <property name="text">
                <string>4</string>
               </property>
              </item>
              <item row="3" column="4">
               <property name="text">
                <string>4000</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QFrame" name="frame_4">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
border-top:2px dotted#111;
border-bottom:2px dotted#111;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">border:0px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="icons.qrc">:/icon/user.png</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border:0px;</string>
           </property>
           <property name="text">
            <string>www.sadan-toshiba.com</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border:0px;</string>
           </property>
           <property name="text">
            <string>اسم المحل  :  بيع قطع غيار </string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_5">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_14">
        <property name="font">
         <font>
          <family>MingLiU_HKSCS</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 170, 0);
color: rgb(255, 255, 255);
padding:10px;
</string>
        </property>
        <property name="text">
         <string>دفع</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icon/mony.png</normaloff>:/icon/mony.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_13">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
padding:10px;
</string>
        </property>
        <property name="text">
         <string>طباعة</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icon/print.png</normaloff>:/icon/print.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QFrame" name="frame_6">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QFrame" name="frame_7">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
